docs/technologies.md

```diff
@@ -4,7 +4,7 @@ BigFlow provides support for the main big data processing technologies on GCP:
 
 * [Dataflow](#dataflow-apache-beam) (Apache Beam)
 * [BigQuery](#bigquery)
-* [Dataproc](#dataproc) (Apache Spark)
+* [Dataproc](dataproc.md) (Apache Spark)
```

Comment:
Append dataproc.md content to this document. It's hard to guess that this link leads to the internal document, and it looks inconsistent (Dataflow and BigQuery don't have separate docs).

Reply:
done